Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Filters.kt

```diff
@@ -75,18 +76,20 @@ fun FilterChip(
     shape: Shape = MaterialTheme.shapes.small
 ) {
     val (selected, setSelected) = filter.enabled
-    val backgroundColor =
-        animate(if (selected) JetsnackTheme.colors.brand else JetsnackTheme.colors.uiBackground)
+    val backgroundColor by
+    animateAsState(if (selected) JetsnackTheme.colors.brand else JetsnackTheme.colors.uiBackground)
```

Comment:
Indentation is slightly off here. Indentation in L86 looks great though, maybe follow the same pattern?

Reply:
done